java-frontend/src/main/java/org/sonar/java/resolve/SemanticModel.java

```diff
@@ -129,6 +130,10 @@ public Symbol getEnclosingClass(Tree tree) {
     return getEnv(tree).enclosingClass;
   }
 
+  public Type getClassType(String fullyQualifiedName) {
```

Comment:
Agreed. I'm relying too much on the fact that `bytecodeCompleter` used to forward the call is fully tested. Fixing it now.